Dockerfile

```diff
@@ -15,6 +15,11 @@ RUN yum install jq -y
 COPY package.json .
 COPY package-lock.json .
 COPY ./scripts/copy-govuk-frontend-assets.sh scripts/copy-govuk-frontend-assets.sh
+COPY ./scripts/copy-moj-frontend-assets.sh scripts/copy-moj-frontend-assets.sh
+
+RUN npm ci && \
```

Comment:
Duplicate

Reply:
Whoops! Thank you, new commit pushed with changes.